# Request
int32 drum_id
int32 drum_pos
---
# No response